SU-SWS/ace-sdssgryphon
# SDSS-369: removed display of shortTitle from person pages

# READY FOR REVIEW

# Summary
- Updated config to remove the display of the shortTitle field from person pages

# Review By (Date)
- ???

# Urgency
- low

# Steps to Test

1. checkout branch
2. `drush cim`
3.  Confirm the field no longer appears above the person's name in people pages


# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
